BEST_PRACTICES.md

```diff
@@ -529,6 +529,18 @@ attendeeList.sort { a, b in
 }
 ```
 
+Always used named parameters, and never unamed ordered parameters (`$0`). It's harder to write, harder to read and takes much longer to compile.
```

Comment:
wow. Is it personal experience or you found it somewhere? Would be awesome to add the source.